test_runner/performance/test_gc_feedback.py

```diff
@@ -33,7 +33,7 @@ def test_gc_feedback(neon_env_builder: NeonEnvBuilder, zenbenchmark: NeonBenchma
             "checkpoint_distance": f"{1024 ** 2}",
             "compaction_target_size": f"{1024 ** 2}",
             # set PITR interval to be small, so we can do GC
-            "pitr_interval": "10 s",
+            "pitr_interval": "60 s",
```

Comment:
Why this change?

Reply:
If the interval is too small, the GC horizon would be above all existing delta+image layers, and the bottom-most compaction simply removes everything and creates image layers. By making the interval larger, we can have some deltas above the GC horizon, therefore covering the full code path in this benchmark.

Reply:
Hm, seems like an ugly source of non-determinism in this test.

Could we instead use `pitr_interval: 1y` and pass the right `gc_horizon` to `/compaction?legacy_enhanced...=yes`, so that the inputs are more deterministic?

Reply:
because the benchmark runs for arbitrary time, it's also hard to pre-determine a gc_horizon for the benchmark in advance... anyways, I feel it's fine to have some non-determinism in benchmarks. The specific cases that gc_horizon cuts through some deltas, or above all deltas, are in Rust unit tests.

Reply:
Fair, it's just that the benchmark results here will vary _massively_ depending on execution time => not really comparable.

But we don't have to solve this now.